tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -499,6 +503,16 @@ def add(self, ticket: SessionTicket) -> None:
     def pop(self, label: bytes) -> Optional[SessionTicket]:
         return self.tickets.pop(label, None)
 
+class WebTransportCertificateGeneration(Enum):
+    """
+    Specify, if the server should generate a certificate or use an existing certificate
+     USEPREGENERATED: use existing certificate
+     GENERATEDVALIDSERVERCERTIFICATEHASHCERT: generate a certificate compatible to server cert hashes
```

Comment:
This makes it a little bit more readable:
```suggestion
     USE_PREGENERATED: use existing certificate
     GENERATE_VALID_SERVER_CERTIFICATE_HASH_CERT: generate a certificate compatible to server cert hashes
```

Reply:
Done.